azure-pipelines.yml

```diff
@@ -210,3 +193,28 @@ stages:
         -TsaRepositoryName "Arcade-Validation"
         -TsaCodebaseName "Arcade-Validation"
         -TsaPublish $True'
+
+  - stage: Push_to_latest_channel
```

Comment:
maybe specify a display name?

Reply:
Yep, that was in my to do list. Thanks for the reminder